Packs/PrismaCloudCompute/README.md

```diff
@@ -1,6 +1,38 @@
-![image](https://user-images.githubusercontent.com/49071222/72906531-0e452a00-3d3b-11ea-8703-8b97ddf30be0.png)
+# Prisma Cloud Compute
+This pack includes Cortex XSIAM content.
 
+<~XSIAM>
+A step-by-step configuration process is available at Cortex XSIAM Administrator Guide- [Ingest Alerts from Prisma Cloud Compute](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Alerts-from-Prisma-Cloud). 
 
+## Configuration on XSIAM
+1. Select Settings → Data Sources.
+2. In the Prisma Cloud Compute Collector configuration, click **Add Instance** to begin a new alerts integration.
+3. Specify the Name for the Prisma Cloud Compute Collector displayed in Cortex XSIAM.
+4. Save & Generate Token. The token is displayed in a blue box, which is blurred in the image below.
+   * Click the Copy icon next to the Username and Password, and record them in a safe place, as you will need to provide them when you configure the Prisma Cloud Compute Collector for alerts integration. If you forget to record the key and close the window, you will need to generate a new key and repeat this process. When you are finished, click Done to close the window.
```

Comment:
This instruction is for saving the token.  What is the instruction for generating the token?

Reply:
When you press on Save the token is auto-generated and presented to the user.